extensions/algebra/guest/src/halo2curves.rs

```diff
@@ -1,55 +1,76 @@
-use core::ops::{Add, Mul, Sub};
-
 use halo2curves_axiom::ff;
 
 use crate::{field::Field, DivAssignUnsafe, DivUnsafe};
 
-impl<'a, F: ff::Field> DivUnsafe<&'a F> for F {
```

Comment:
it doesn't really matter, but what stopped you from doing a blanket implementation for all `ff::Field`?